src/app/clients/add/formSections/NumberAdultsCard.tsx

```diff
@@ -13,7 +13,14 @@ import {
 } from "@/components/Form/formFunctions";
 import GenericFormCard from "@/components/Form/GenericFormCard";
 import { selectChangeEventHandler } from "@/components/DataInput/inputHandlerFactories";
-import { GappedDiv } from "@/components/Form/formStyling";
+
+const getNumberAdultsDefault = (adults: Person[], gender: string): string | undefined => {
+    const personIndex = adults.findIndex((person) => person.gender === gender);
+    return adults[personIndex].quantity === 0
+        ? undefined
```

Comment:
Just want to check 
What if it is actively being put as a gender being 0 in a household - do you still want to display nothing or you want to explicitly display it is 0

Reply:
Or it is just non differentiable in our current database because all number are initialized as 0 and there is no "null"?

Reply:
Would explicitly showing it is 0 important though - otherwise people might thought it is just not filled rather than assuming maybe a household has no specific gender

Reply:
0 and not filled are basically the same in this case, because it defaults to 0. And the form makes it so that there will be at least one adult in the entire household. And 0 means there won't be any records added to the database

Reply:
do let me know if you need more explanation